diff -Nur postfix-3.2.3-orig/conf/post-install postfix-3.2.3/conf/post-install
--- postfix-3.2.3-orig/conf/post-install	2015-12-27 19:00:45.000000000 -0500
+++ postfix-3.2.3/conf/post-install	2017-10-20 11:15:08.818396118 -0400
@@ -430,6 +430,7 @@
 # someone really wants the expanded mail version in main.cf, then
 # we're sorry.
 
+
 # Confine side effects from mail_version unexpansion within a subshell.
 
 (case "$mail_version" in
@@ -532,7 +533,17 @@
 	    case $path in
 	    no|no/*) continue;;
 	    esac
-	    # Pick up the flags.
+	    # Munge paths for alternatives.
+	    case $path in
+      		/usr/bin/mailq) path=$path.postfix ;;
+      		/usr/bin/newaliases) path=$path.postfix ;;
+      		/usr/bin/rmail) path=$path.postfix ;;
+      		/usr/sbin/sendmail) path=$path.postfix ;;
+      		/usr/share/man/man1/mailq.1.gz) path=/usr/share/man/man1/mailq.postfix.1.gz ;;
+      		/usr/share/man/man1/newaliases.1.gz) path=/usr/share/man/man1/newaliases.postfix.1.gz ;;
+      		/usr/share/man/man5/aliases.5.gz) path=/usr/share/man/man5/aliases.postfix.5.gz ;;
+            esac
+            # Pick up the flags.
 	    case $flags in *u*) upgrade_flag=1;; *) upgrade_flag=;; esac
 	    case $flags in *c*) create_flag=1;; *) create_flag=;; esac
 	    case $flags in *r*) recursive="-R";; *) recursive=;; esac
